### PR TITLE
change button styling in prep to support new function + other minor f…

### PR DESCRIPTION
**Pull request by miasya**
…ixes

### CODE REVIEW

File: portfolio/src/main/webapp/recipes/recipe-1.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          <a href="javascript:void(0);" class="icon" onclick="navBar()">
+            <i class="fa fa-bars"></i>
+          </a>
+        </div>
+      </head>
+
+      <body>
+        <h1>Recipe Name</h1>
+        <img src="../images/eggrolls.jpg" alt="Food" width="80%"></img>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
```
Comment (lipczak-g) [inline portfolio/src/main/webapp/recipes/recipe-1.html:29]: Please avoid submitting test code (or test data in code) to the main branch. This is risky as it makes it much easier to get to production this way. Either remove it before the submission or put it in a "experimental" file that stays in your branch.

Comment (miasya) [reply portfolio/src/main/webapp/recipes/recipe-1.html:29]: Are you referring to Lorem ipsum? If so I can populate that with actual text in an upcoming commit

**Comment by miasya** [comment]
Recipe image: https://screenshot.googleplex.com/jndztQuSDd7.png

**Comment by miasya** [comment]
Top of main page image: https://screenshot.googleplex.com/kS0EKinavAE.png
Bottom of main page image: https://screenshot.googleplex.com/KGig1o1z50D.png

**Comment by miasya** [comment]
Recipe card image: 
image

**Comment by miasya** [comment]
Image of word definition card on contact page: 
image

File: portfolio/src/main/webapp/recipes/recipe-1.html (hunk 1 above)
Comment (lipczak-g) [reply portfolio/src/main/webapp/recipes/recipe-1.html:29]: yes, "Lorem ipsum" here, but test strings in general. I think "whatever goes into master will end up in production" is a good rule of thumb for this.

Comment (miasya) [reply portfolio/src/main/webapp/recipes/recipe-1.html:29]: Ok got it!

**Comment by miasya** [comment]
Image of contact page: 
image

**Pull request merged**